File: wallofslaughter/a_murk_hatchling.pl
sub EVENT_SPAWN {
  quest::emote("drips a green poison from its maw as it approaches.");
  quest::settimer("depop_hatchling",600);
}

sub EVENT_TIMER {
  if($timer == "depop_hatchling") {
    quest::depop();
  }
}

File: wallofslaughter/a_murk_hatchling.lua
function event_spawn(e)
eq.set_timer("depop", 600 * 1000);
e.self:Emote("drips a green poison from its maw as it approaches.");
end

function event_combat(e)
if(e.joined) then
eq.stop_timer("depop");
else
eq.set_timer("depop", 600 * 1000);
end
end

function event_timer(e)
if (e.timer == "depop") then
eq.depop();
end
end
